docs/core-concepts/introduction.md

```diff
@@ -11,7 +11,7 @@ Karmada (Kubernetes Armada) is a Kubernetes management system that enables you t
 Karmada aims to provide turnkey automation for multi-cluster application management in multi-cloud and hybrid cloud scenarios,
 with key features such as centralized multi-cloud management, high availability, failure recovery, and traffic scheduling.
 
-Karmada is a sandbox project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF).
+Karmada is an incubation project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF).
```

Comment:
nice job~ there is a same statement in [karmada repo](https://github.com/karmada-io/karmada/blob/master/README.md?plain=1#L23) but mistakenly used 'an' to 'a', could you please modify it together? Thanks

Reply:
Got it